src/apps/vpn/platforms/android/androidvpnactivity.cpp

```diff
@@ -160,3 +163,19 @@ void AndroidVPNActivity::startAtBootChanged() {
 void AndroidVPNActivity::onLogout() {
   sendToService(ServiceAction::ACTION_CLEAR_STORAGE);
 }
+
+void AndroidVPNActivity::onAppStateChange() {
+  if (!Constants::inProduction()) {
+    // Do not restrict screencap on debug
+    return;
+  }
```

Comment:
>  get it might be good for QA 
We usually do not enable security things on debug - given it is a debug build. Like qml debugging, inspector etc. I do not see why this would be cased diffrently :)